industrial_ci/src/builders/colcon.sh

```diff
@@ -28,7 +28,11 @@ function builder_setup {
 function builder_run_build {
     local extend=$1; shift
     local ws=$1; shift
-    local opts=(--event-handlers "${_colcon_event_handlers[@]}")
+    local output_handler
+    if [ "${VERBOSE_OUTPUT:-false}" != false ]; then
+        output_handler="console_cohesion+"
+    fi
+    local opts=(--event-handlers "${_colcon_event_handlers[@]}" "${output_handler}")
```

Comment:
This will not work properly, because it adds an empty string, if VERBOSE_OUTPUT=false

Please try
```suggestion
    local opts=(--event-handlers "${_colcon_event_handlers[@]}")
    if [ "${VERBOSE_OUTPUT:-false}" != false ]; then
        opts+(="console_cohesion+")
    fi
```

Reply:
Thanks :) Commited the change (with small typo fix) and tested it in our internal repo, still worked as expected.

Reply:
Ah, this got hidden already ;)
Just ignore my other comment..